### ISSUE TITLE
Discord embeds have strange spacing when linking pages

### ISSUE THREAD

**Issue by smashmath** [alshedivat/al-folio#493]
**Acknowledge the following**
- [x] I carefully read and followed the [Getting Started](https://github.com/alshedivat/al-folio#getting-started) guide.
- [x] I read through [FAQ](https://github.com/alshedivat/al-folio#faq) and searched through the [past issues](https://github.com/alshedivat/al-folio/issues), none of which addressed my issue.
- [x] The issue I am raising is a potential bug in al-folio and not just a usage question. <br> [For usage questions, please post in the [Discussions](https://github.com/alshedivat/al-folio/discussions) instead of raising an issue.]

**Describe the bug**
When sharing links to sites using the theme (other than the about page), there is some strange spacing between the site name and page title. Example:
![](https://i.imgur.com/j7V1Dsa.png)

**To Reproduce**
1. Go to discord (desktop client, browser, app, it doesn't matter)
2. Paste any link to a page using the theme (other than the about page) into a message

**Expected behavior**
To look like it did before without a giant space between the site name and page title.

**Screenshots**
This is how embeds used to look vs how they look now. I noticed this first because I had edited the second message which had a link in it today after updating the theme.
(Despite being Nov 26, it had been several months since I had updated to the most current version of al-folio). This bug must have cropped up sometime in the last few months.
![](https://i.imgur.com/EYxIF3j.png)

**System (please complete the following information):**
This stuff is kinda irrelevant since it's on the discord client, browser instances of discord, and on the discord mobile app rather than the site itself, but whatever.
 - OS: Windows 10
 - Browser (and its version) Chrome 96.0.4664.45
 - Jekyll version 4.2.1
- Ruby version 3.0

**Additional context**
This is a fresh install and not unique to my blog.

**Comment by jmrplens** [comment]
Solved with my PR #481 

In the meantime, you can change the title block in `_includes/head.html` to this:

```html

<title>
{% if site.title == "blank" %}
{% capture title %}{{ site.first_name }} {{ site.middle_name }} {{ site.last_name }}{% endcapture %}
{% else %}
{% capture title %}{{ site.title }}{% endcapture %}
{% endif %}
{% if page.title != "blank" and page.url != "/" %}
{{ title }} | {{ page.title }}
{% else %}
{{ title }}
{% endif %}
</title>
```

**Comment by smashmath** [comment]
@jmrplens This fixed it. Thank you!
